metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -787,14 +806,18 @@ def merge_refinement(raw_view: dict, refinement_views: List[dict]) -> dict:
 
         return new_raw_view
 
-    def get_refinement_from_looker_view_file(
-        self, looker_view_file: LookerViewFile, view_name: str
+    def get_refinement_from_view_list(
+        self, views: List[dict], view_name: str
     ) -> List[dict]:
-        refined_view: str = self.REFINEMENT_PREFIX + view_name
+        """
+        Refinement syntax for view and explore are same.
```

Comment:
I find it extremely confusing that this is called `_from_view_list` but it works for both explores and views - maybe we can rename to something more generic?

Reply:
I renamed it to `get_refinements`